diff --git a/modules/entity_share_client/src/Entity/Remote.php b/modules/entity_share_client/src/Entity/Remote.php
index 3ca8080..0594806 100644
--- a/modules/entity_share_client/src/Entity/Remote.php
+++ b/modules/entity_share_client/src/Entity/Remote.php
@@ -39,7 +39,6 @@ use Drupal\entity_share_client\ClientAuthorization\ClientAuthorizationInterface;
  *     "auth",
  *   },
  *   links = {
- *     "canonical" = "/admin/config/services/entity_share/remote/{remote}",
  *     "add-form" = "/admin/config/services/entity_share/remote/add",
  *     "edit-form" = "/admin/config/services/entity_share/remote/{remote}/edit",
  *     "delete-form" = "/admin/config/services/entity_share/remote/{remote}/delete",
